docker/apm-server/Dockerfile

```
@@ -1,7 +1,13 @@
 ARG apm_server_base_image=docker.elastic.co/apm/apm-server:8.0.0-SNAPSHOT
 ARG go_version=1.13
+ARG apm_server_binary=apm-server
+
+###############################################################################
+# Build stage: build apm-server binary and update apm-server.yml
+###############################################################################
 
 FROM golang:${go_version} AS build
+ARG apm_server_binary
```

by default, we can build the x-pack version

```suggestion
ARG apm_server_binary="apm-server"
```

---

The default value comes from the global ARG at the top of the Dockerfile - it's not necessary to repeat it.

---

(It is necessary to have an `ARG apm_server_binary` in each build stage though: https://github.com/moby/moby/issues/37345)